pwndbg/gdblib/kernel/__init__.py

```diff
@@ -165,6 +168,9 @@ def __init__(self) -> None:
         self.PAGE_SHIFT = 12
         self.phys_base = 0x1000000
 
+    def page_size(self) -> int:
+        return 1 << self.PAGE_SHIFT
```

Comment:
Is `PAGE_SHIFT` constant? I believe so and so we should make it a constant of the module or a class variable. Then, `PAGE_SIZE` should probably also be a constant.

We also have `pwndbg.gdblib.memory.PAGE_SIZE = 0x1000`, maybe we should just use it here ( ? ).

Reply:
(Otherwise, if we keep it as it is now, each call to `.page_size()` will recompute the shift; not a big deal, but still)

Reply:
Hmm I can see that we may set `PAGE_SHIFT` based on a config:
```py
self.PAGE_SHIFT = int(kconfig()["CONFIG_ARM64_PAGE_SHIFT"])
```
 
So it may change (?)

Maybe we should just move the `page_size` impl to base class since both children have the same impl? Or do we want to protect against incorrect impl on a new arch the way its done right now?